packages/svg-serializer/index.js

```diff
@@ -1,38 +1,9 @@
-// import { CSG } from '@jscad/csg'
 const {CSG} = require('@jscad/csg')
 const stringify = require('onml/lib/stringify')
 
 const mimeType = 'image/svg+xml'
 
-function dPath (path, xoffset, yoffset) {
-  var pointindex
-  var str = ''
-  var numpointsClosed = path.points.length + (path.closed ? 1 : 0)
-  for (pointindex = 0; pointindex < numpointsClosed; pointindex++) {
-    var pointindexwrapped = pointindex
-    if (pointindexwrapped >= path.points.length) pointindexwrapped -= path.points.length
-    var point = path.points[pointindexwrapped]
-    if (pointindex > 0) {
-      str += 'L' + (point.x + xoffset) + ' ' + (point.y + yoffset)
-    } else {
-      str += 'M' + (point.x + xoffset) + ' ' + (point.y + yoffset)
-    }
-  }
-  return str
-}
-
-function PathsToSvg (paths, bounds, options) {
-  // calculate offsets in order to create paths orientated from the 0,0 axis
-  var xoffset = 0 - bounds[0].x
-  var yoffset = 0 - bounds[0].y
-
-  return paths.reduce(function (res, path, i) {
-    options && options.statusCallback && options.statusCallback({progress: 100 * i / paths.length})
-    return res.concat([['path', {d: dPath(path, xoffset, yoffset)}]])
-  }, ['g'])
-}
-
-function serialize (cagObject, options) {
+const serialize = function (cagObject, options) {
```

Comment:
Very good call, having the serialize function first is way better!
Side note : at some later point we really need to swap the order of options, cagObject, but that would be a breaking change for all of them , so not urgent :)